code/modules/fishing/fish/types/rift.dm

```diff
@@ -400,3 +400,427 @@
+
+	visible_message(span_suicide("[user] puts [src] against their lips! It looks like they're preparing to say something!"))
+	var/psychic_speech = tgui_input_text(user, message = "Say something!", title = "What are your last words?", timeout = 15 SECONDS)
+	if(!psychic_speech || !locate(src) in user.get_contents())
+		user.say("Err, umm... uhh... erm...", forced = "blustering like a moron due to babbelfish suicide")
+		visible_message(span_suicide("[user] dies from shame!"))
+		return OXYLOSS
+
+	voice_of_god(psychic_speech, user, list("big", "alertalien"), base_multiplier = 5, include_speaker = TRUE, forced = TRUE, ignore_spam = TRUE)
+	psy_wail()
+	user.adjustOrganLoss(ORGAN_SLOT_BRAIN, INFINITY, INFINITY, ORGAN_SLOT_BRAIN)
+	user.death()
+	return MANUAL_SUICIDE
+
+/**
+ * When the fish dies you die in real life.
+ * Consequently, when the fish is magically resuscitated the bound mob is revived as well.
+ */
+/obj/item/fish/babbelfish/proc/fishes_die_twice()
+	if(isnull(moron_inside))
+		UnregisterSignal(src, COMSIG_FISH_STATUS_CHANGED)
+	if(status == FISH_DEAD)
+		moron_inside.death()
+	if(status == FISH_ALIVE)
+		moron_inside.revive(HEAL_ALL)
+
+/**
+ * If they somehow escaped null the variable.
+ */
+/obj/item/fish/babbelfish/proc/check_loc()
+	if(locate(moron_inside) in src)
+		return
+	UnregisterSignal(moron_inside, COMSIG_MOVABLE_MOVED)
+	moron_inside = null
+
+/obj/item/fish/babbelfish/set_status(new_status, silent)
+	. = ..()
+	// Death message plays here
+	if(new_status != FISH_DEAD)
+		damtype = BRAIN
+		return
+	psy_wail()
+	damtype = BRUTE
+
+/**
+ * When a babbelfish dies, it will let out a wail that kills any fish nearby, alongside severely incapacitating anyone else around.
+ * This is punishment for neglecting your catches.
+ */
+/obj/item/fish/babbelfish/proc/psy_wail()
+	manual_emote("wails!")
+	playsound(src, 'sound/mobs/non-humanoids/fish/fish_psyblast.ogg', 100)
+	var/list/mob/living/mobs_in_range = get_hearers_in_range(7, src)
+	for(var/mob/living/screeched in mobs_in_range)
+		if(screeched.can_block_magic(MAGIC_RESISTANCE_MIND, charge_cost = 1))
+			to_chat(screeched, span_notice("You resist the psychic wail!"))
+			continue
+		var/power = 1
+		if(!screeched.can_hear()) // bit weaker if deaf. but its still psychic
+			power *= 0.5
+		var/affect_time = 15 SECONDS * power
+		// it really fucks you up
+		screeched.Knockdown(affect_time)
+		screeched.adjust_disgust(affect_time)
+		screeched.adjust_stutter(affect_time)
+		screeched.adjust_slurring(affect_time)
+		screeched.adjust_dizzy(affect_time)
+		screeched.adjust_staggered(affect_time)
+		screeched.adjust_jitter(affect_time)
+		screeched.adjust_confusion(affect_time)
+		screeched.adjust_hallucinations(affect_time)
+		screeched.adjust_eye_blur(affect_time)
+		if(iscarbon(screeched))
+			var/mob/living/carbon/carbon_screeched = screeched
+			carbon_screeched.vomit()
+			carbon_screeched.adjustOrganLoss(ORGAN_SLOT_BRAIN, 50)
```

Comment:
You're just asking for someone with a tinfoil hat to bring four of them to medbay or the brig and take a bite or two off one of them (the others dulily dropped on the floor) to unexpectedly insta-kill every other human on screen without warning at this point.
The question is how long till it happens and how hard or easy it will be to get it done, so I'm knowingly going to let this slide until that happens, think of this as a memo though.